radiation/radiation_two_stream.F90

```diff
@@ -86,7 +86,9 @@ subroutine calc_two_stream_gammas_lw(ng, ssa, g, &
 
     if (lhook) call dr_hook('radiation_two_stream:calc_two_stream_gammas_lw',0,hook_handle)
 #endif
+    !$acc routine worker
 
+    !$acc loop independent worker
```

Comment:
@FrostyMike is there a reason why you use worker and not vector for the inner most loop ?

Reply:
Yes, check the old message in slack channel for details.
"2. Compiler (pgfortran 20.11) does not respect OpenACC "vector_length" specification.
This means that Gang -> Vector parallelism model I outlined in my variants is not going to work as is. However, this can be easily fixed by introducing workers, in the amount required to fill the vector. I.e. if you want 1 vector of 128 threads, create 4 workers with 32 threads each instead."